geomesa-tools/bin/install-hadoop-accumulo.sh

```diff
@@ -7,6 +7,21 @@
 # http://www.opensource.org/licenses/apache2.0.php.
 #
 
+# This script will attempt to install the client dependencies for hadoop and accumulo
+# into a given directory. Usually this is used to install the deps into either the
+# geomesa tools lib dir or the WEB-INF/lib dir of geoserver.
+
+accumulo_version="${accumulo.version}"
+hadoop_version="${hadoop.version}"
+zookeeper_version="${zookeeper.version}"
+thrift_version="${thrift.version}"
+
+# for hadoop 2.5 and 2.6 to work we need these
+guava_version="11.0.2"
+com_log_version="1.1.3"
```

Comment:
2.4 uses the same guava/commons as well...anything further back than that i think we'd have issues with the CL tools with Classpath anyways.